tests/Integration/Behaviour/Features/Scenario/Order/order_add_payment.feature

```diff
@@ -0,0 +1,136 @@
+    And I add 2 products "Mug The best is yet to come" to the cart "dummy_cart"
+    And I add order "bo_order1" with the following details:
+      | cart                | dummy_cart                 |
+      | message             | test                       |
+      | payment module name | dummy_payment              |
+      | status              | Awaiting bank wire payment |
+
+# 4 cases
+#
+#    Order is in default_currency + Payment is in Order currency
+#       for example default = 1, order = 1, payment = 1
+#       ==> NO conversion to do
+#    Order is in default_currency + Payment is NOT in Order currency
+#       for example default = 1, order = 1, payment = 2
+#       ==> convert payment in order's currency
+#    Order is NOT in default_currency + Payment is in Order currency
+#       for example default = 1, order = 2, payment = 2
+#       ==> NO conversion to do
+#    Order is NOT in default_currency + Payment is NOT in Order currency
+#       for example default = 1, order = 2, payment = 3
+#       ==> As conversion rates are set regarding the default currency,
+#           convert payment to default and from default to order's currency
+
+  Scenario: Add a payment when Order is in default_currency and Payment is in Order currency
+    When order "bo_order1" has 0 payments
+    When I pay order "bo_order1" with the following details:
+      | date           | 2019-11-26 13:56:23 |
+      | payment_method | Payments by check   |
+      | transaction_id | test123             |
+      | currency       | USD                 |
+      | amount         | 6.00                |
+    Then order "bo_order1" payments should have the following details:
+      | date           | 2019-11-26 13:56:23 |
+      | payment_method | Payments by check   |
+      | transaction_id | test123             |
+      | amount         | $6.00               |
+    And order "bo_order1" should have the following details:
+      | total_paid_real           | 6.000000 |
+
+  Scenario: Add a payment when Order is in default_currency and Payment is NOT in Order currency
+    When order "bo_order1" has 0 payments
+    When I pay order "bo_order1" with the following details:
+      | date           | 2019-11-26 13:56:23 |
+      | payment_method | Payments by check   |
+      | transaction_id | test123             |
+      | currency       | currency2           |
+      | amount         | 6.00                |
+    Then order "bo_order1" payments should have the following details:
+      | date           | 2019-11-26 13:56:23 |
+      | payment_method | Payments by check   |
+      | transaction_id | test123             |
+      | amount         | €6.00               |
+    And order "bo_order1" should have the following details:
+      | total_paid_real           | 6.820000 |
+
+#  Scenario: Add a payment when Order is NOT in default_currency and Payment is in Order currency
```

Comment:
Are these commented scenarios useful? :thinking:

Reply:
They are commented in order to fix them for the next release or if we have time to fix after the Beta